API.md

```diff
@@ -3,48 +3,55 @@
 This is a JavaScript classes documentation which describes both client and server instance creation and management.
 
 #### Table of Contents
-* [Client](#client)
```

Comment:
Sorry, what happened to these lines?
The dataPack part looks good.

Reply:
Sorry, it may be that I copied the extra content due to a mistake when editing the document. I've removed the redundant content in the latest commit.